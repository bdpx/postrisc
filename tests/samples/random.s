.text
    write	"test random"
    alloc	96

    random	%r3, %gz
    write	"random: %x64(r3)"
    random	%r3, %gz
    write	"random: %x64(r3)"
    ldi		%r4, 1
    random	%r3, %r4
    write	"random seed: %x64(r3)"

    write	"end_random_test"
.end
