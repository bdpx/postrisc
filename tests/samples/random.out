test random
random: 086b1d783f0b36f8
random: a218dbbc13a36bcd
random seed: 2b7284fe29291b48
end_random_test
